src/CheckersBoard.java

```diff
@@ -1,553 +1,403 @@
+		return true;
+	    }
+	    else {
+		return false;
+	    }
+	}
+	else {
+	    if (i == 7 && (j == 0 || j == 2 || j == 4 || j == 6)) {
+		return true;
+	    }
+	    else {
+		return false;
+	    }
+	}
+    }
+    
+    public void checkWinner() {
+	if (xCount == 0) {
+	    winner = 'x';
+	}
+	else if (oCount == 0) {
+	    winner = 'o';
+	}
+    }
+    
+    public void move(int fromI, int fromJ, int toI, int toJ) {
+	setMoves(fromI, fromJ, toI, toJ);
+	validMove(fromI, fromJ, toI, toJ);
+	if (canJump == true && validMove == true) {
+	    pieces[fromI][fromJ] = ' ';
+	    if (kingCheck(toI, toJ)) {
+		if (turn == 'x') {
+		    pieces[toI][toJ] = 'X';
+		    pieces[jumpedI][jumpedJ] = ' ';
+		    --oCount;
+		}
+		else {
+		    pieces[toI][toJ] = 'O';
+		    pieces[jumpedI][jumpedJ] = ' ';
+		    --xCount;
+		}
 	    }
+	    else {
+		pieces[toI][toJ] = turn;
+		pieces[jumpedI][jumpedJ] = ' ';
+	    }
+	}
+	if (canJump == false && validMove == true) {
+	    pieces[fromI][fromJ] = ' ';
+	    pieces[toI][toJ] = turn;
+	}
+	if (validMove == false) {
+	    // throw new CheckersIllegalMoveException("Please make a valid move");
+	    error = true;
+	}
+	validMove = false;
+	canJump = false;
+    }
+    
 
-	    Spre = String.valueOf(Cpre);
-	    Ssuf = String.valueOf(Csuf);
+    /**A toString method for the CheckersBoard, allows easier access for console based interface
+     * @return result The result of the toString(), aka the gameBoard
+     */
+    public String toString() {
+	String result;
+	result = " | A | B | C | D | E | F | G | H |";
 
-	    Spre = Spre.toUpperCase();
-	    
-	    r = Spre + Ssuf;
+	for (int i = 0; i < 8; i++) {
+	    result += "\n-+---+---+---+---+---+---+---+---+\n";
+	    result += Integer.toString(i + 1) + "|";
+	    for (int j = 0; j < 8; j++) {
+		result += " " +  pieces[i][j] + " |";
+	    }
+	}
+	result += "\n-+---+---+---+---+---+---+---+---+\n";
+	return result;
+    }
 
-	    return r;
+    /** A formatting function that allows multiple different types of inputs
+     * Matt already completed?
+     * @param input from user, either piece theyre selecting or spot they want to move to
+     * @return input in proper format
+     */
+    public String cleanCoord(String s) {
+	String r = s, Spre, Ssuf;
+	char Cpre, Csuf;
+	// S isn't 2 chars, invalid, return entry
+	if (r.length() != 2) return s;
+	Cpre = r.charAt(0);
+        Csuf = r.charAt(1);
+
+            // First make sure that format is <Letter><Digit>
+        if (Character.isDigit(Cpre) && Character.isLetter(Csuf)) { // Format is <Digit><Letter>, switch it     
+            char temp = Csuf;
+            Csuf = Cpre;
+            Cpre = temp;
 	}
-}//end class Checkers
+        Spre = String.valueOf(Cpre);
+        Ssuf = String.valueOf(Csuf);
+	Spre = Spre.toUpperCase();
+        r = Spre + Ssuf;
+        return r;
+    }
+	
+    public int findYCoordinate(String s) {
+	char coord = s.charAt(0);
+	int x;
+	if ( coord == 'A') {x = 0; }
```

Comment:
You can do this easier.  Make sure it's between 'A' and 'H' and then subtract 'A'.